engine/security/https.md

```diff
@@ -82,13 +82,22 @@ using `10.10.10.20` and `127.0.0.1`:
 
     $ echo subjectAltName = DNS:$HOST,IP:10.10.10.20,IP:127.0.0.1 > extfile.cnf
 
+In any case, set the key's extended usage attributes, so if it is compromised
```

Comment:
maybe: "Set the Docker daemon's key's extended usage attributes to only be used for server authentication:" ?